src/Volumes.php

```diff
@@ -14,30 +16,40 @@ public function __construct(GoogleBooks $client)
 
     /**
      * @param $query
-     * @return \Generator|Volume
+     * @return \Generator|Volume || array
      */
     public function search($query)
     {
+        return (new LibraryBuilder($this->client))->search($query);
+    }
+
+    /**
+     * @param $query
+     * @return \Generator|Volume
```

Comment:
No longer Generator (btw. phpdoc doesn't have good support for documenting generators as you see)

Reply:
ill change this